aws_sagemaker_studio/sagemaker_experiments/mnist-handwritten-digits-classification-experiment.ipynb

```diff
@@ -307,7 +307,7 @@
     "    # all input configurations, parameters, and metrics specified in estimator \n",
     "    # definition are automatically tracked\n",
     "    estimator = PyTorch(\n",
-    "        entry_point='./mnist.py',\n",
+    "        entry_point='aws_sagemaker_studio/sagemaker_experiments/mnist.py',\n",
```

Comment:
most of the other notebooks [use the current directory style](https://github.com/awslabs/amazon-sagemaker-examples/search?l=Jupyter+Notebook&p=3&q=entry_point) and we'd like to keep it that way for consistency.  you could add an [ENTRY_POINT](https://github.com/awslabs/amazon-sagemaker-examples/blob/5a898eb601a21f4bb764afa0fc7b772e28cb91be/sagemaker-python-sdk/dgl_kge/kge_pytorch.ipynb) variable instead though.

Reply:
I can add that at a later date. I reverted the change to just amend the typo